src/components/DataTable.tsx

```diff
@@ -210,6 +226,8 @@ export function DataTable<T extends object>(props: Props<T>) {
       })
       .finally(() => {
         queryClient.invalidateQueries([props.endpoint]);
```

Comment:
I leave this line of code because of some potential queries that include query parameters in the first query key while invalidating (`/api/v1/designs?status=active...` instead of just `/api/v1/designs`).